src/components/Form/FormSection.jsx

```diff
@@ -27,7 +27,8 @@ const FormSection = ({ title, headingId, children }) => (
       alignItems: 'center',
       mx: '3px',
       padding: '16px',
-      minWidth: '50%'
+      minWidth: '50%',
+      maxWidth: '75vdw'
```

Comment:
Minor thing, but I believe you mean the `dvw` unit here for device viewport width?